content/en/news/releases/1.6.x/announcing-1.6/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Announcing Istio 1.6
+linktitle: 1.6
+subtitle: Major Update
+description: Istio 1.6 release announcement.
+publishdate: 2020-05-19
+release: 1.6.0
+skip_list: true
+aliases:
+    - /news/announcing-1.6.0
+    - /news/announcing-1.6
+---
+
+We are pleased to announce the release of Istio 1.6!
+
+{{< relnote >}}
+
+With this release, we are continuing the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, you'll find that Citadel, the sidecar
+injector and Galley are no longer deployed.
+
+You'll also find that we are using the new `AppProtocol` API from Kubernetes.
+What does that mean for you? It means that you won't need to use the name
+field in your service to denote the protocol -- a simpler user experience.
+
+## Better lifecycle
+
+We continue to make installing and upgrading Istio a better experience. Our
+command line tool `istioctl` gives better diagnostic information, has a simpler
+install command, and even gives status in color!
+
+Upgrading Istio has been improved as well, in several powerful ways. First, we
+now support canarying of the Istio control plane itself. That means you can
```

Comment:
How about putting a link to @howardjohn's blog post here?

Reply:
It's not published yet...happy to enlinkify it when that's published.

Reply:
It's in preliminary, targeted to go out with the release. https://preliminary.istio.io/blog/2020/multiple-control-planes/